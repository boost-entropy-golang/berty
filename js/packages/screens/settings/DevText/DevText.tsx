import React from 'react'
import { ScrollView } from 'react-native'

import { UnifiedText } from '@berty/components/shared-components/UnifiedText'
import { ScreenFC } from '@berty/navigation'

export const DevText: ScreenFC<'Settings.DevText'> = ({
	route: {
		params: { text },
	},
}) => {
	return (
		<ScrollView>
			<UnifiedText selectable={true}>{text}</UnifiedText>
		</ScrollView>
	)
}
