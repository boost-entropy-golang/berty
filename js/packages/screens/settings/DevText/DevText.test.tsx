import { renderScreen } from '@berty/utils/testing/renderScreen.test'

import { DevText } from './DevText'

test('Settings.DevText renders correctly', () => {
	const { toJSON } = renderScreen('Settings.DevText', DevText, { text: 'Hello World' })
	expect(toJSON()).toMatchSnapshot()
})
