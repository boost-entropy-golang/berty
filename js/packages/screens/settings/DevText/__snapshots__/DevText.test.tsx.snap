// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings.DevText renders correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <RCTScrollView>
      <View>
        <Text
          selectable={true}
          style={
            Array [
              Object {
                "fontSize": 30,
              },
              Object {
                "color": "#383B62",
                "fontFamily": "Open Sans",
              },
              undefined,
            ]
          }
        >
          Hello World
        </Text>
      </View>
    </RCTScrollView>
  </View>
</RNCSafeAreaProvider>
`;
